apple/testing/default_runner/ios_xctestrun_runner.template.sh

```diff
@@ -123,6 +127,11 @@ if [[ -n "${command_line_args:-}" ]]; then
   xctestrun_cmd_line_args_section="    <key>CommandLineArguments</key>\n    <array>$xctestrun_cmd_line_args_section    </array>"
 fi
 
+xctestrun_attachment_lifetime_section+="    <key>SystemAttachmentLifetime</key>\n"
+xctestrun_attachment_lifetime_section+="    <string>$attachment_lifetime</string>\n"
+xctestrun_attachment_lifetime_section+="    <key>UserAttachmentLifetime</key>\n"
+xctestrun_attachment_lifetime_section+="    <string>$attachment_lifetime</string>"
+
```

Comment:
Can you move this inside the `if [[ "$should_use_xcodebuild" == true ]]; then` block since it's only used there?